src/__tests__/standalone-test.js

```diff
@@ -166,6 +166,27 @@ test("standalone with input css and quiet mode", t => {
   t.plan(planned)
 })
 
+test("standalone with extration from html", t => {
```

Comment:
Can we clarify this description? Am I right that it could be "... extractStyleTagsFromHtml option with straight CSS input"?